### ISSUE TITLE
404 on response from get_fedi_instances()

### ISSUE THREAD

**Issue by iaine** [schochastics/rtoot#122]
When I run get_fedi_instances(n = 5) using root 0.2.0, the response is:

Error in make_get_request(token = NULL, path = "/api/instances", instance = "api.index.community",  : 
  something went wrong. Status code: 404

From what I can see, api.index.community is responding with a 404 but is there a different server to get instances from? If so, is there a way of overriding instances variable in get_fedi_instances() ?

I am running something similar to:
library(rtoot)

instances <- get_fedi_instances(n = 5)
instances$name

**Comment by chainsawriot** [comment]
It's gone

https://api.index.community/

**Comment by schochastics** [comment]
hmm, I will look for a viable alternative. instances.social seems the best

**Comment by chainsawriot** [comment]
Or simply go here: https://instances.social/list/old

**Comment by schochastics** [comment]
True, but I will still implement this function now as an API call with token to not use any screen scraping

**Comment by schochastics** [comment]
@chainsawriot would you be fine with not having a test for the new version of `get_fedi_instances()` using instances.social or is it ok to require devs to setup two env tokens if they want to write tests?

**Comment by chainsawriot** [comment]
@schochastics 
My opinion is that it would be better to have some vcr tests for test coverage.
Maybe create a new envvar called `RTOOT_INSTANCES_SOCIAL_TOKEN` and skip the the tests if it is "".
